tests/console/yast2_dns_server.pm

```diff
@@ -99,41 +56,47 @@ sub run {
             after_reboot  => {start_on_boot => 'alt-a'}
         );
     }
-    send_key 'alt-f';
+    send_key $cmd{finish};
+
+    # Verify changes in console. Apply changes is not availaible to check in the gui first,
+    # Service may take a bit to actually start -> acceptable behaviour (boo#1093029)
     assert_screen 'root-console';
-    # The wizard-like interface still uses the old approach of always starting the service
-    # while enabling it, so named should be both active and enabled
-    $self->assert_running(1);
-    $self->assert_enabled(1);
+    my $times = RETRIES;
+    ($times-- && sleep 5) while (systemctl('is-active named') == 1 && $times);
+    die 'named service is not active even after 25s delay' unless $times;
```

Comment:
I would not call that a delay as we actually check for the service during the waiting time. How about "even after waiting 25s"?